.github/workflows/clang-format-validation.yml

```diff
@@ -1,8 +1,9 @@
 name: Validate code format with clang-format
 on: [push, pull_request]
+
 jobs:
   validate:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
Maybe `[self-hosted, linux]` (per [doc](https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-default-labels-to-route-jobs))?

Reply:
Just re-run resolved the failed CI. But I'll fix it. Thanks.